common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala

```diff
@@ -129,7 +130,7 @@ object CuratedRecipe {
       ),
       published = Some(
         ChangeRecord(
-          date = r.publicationDate.toInstant.toEpochMilli,
+          date = DateTime.now.toInstant.getMillis,
```

Comment:
No, it was a request from @mchv that we set it to the current time. (At least that's what I understood...)